templates/helpers.js

```diff
@@ -240,8 +241,7 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
                 return false;
             }
             var name = packageObject.name,
-                version = 'v' + packageObject.package.version,
-                srcPath = current.src.path.replace('node_modules/' + name + '/', ''),
+                srcPath = current.src.path.replace(SRC_PREFIX_REGEX, ''),
```

Comment:
Can you describe why this change was necessary?  What did `current.src.path` look like?